terraform/loadbalancer.tf

```diff
@@ -0,0 +1,137 @@
+resource "aws_lb" "keyconjurer" {
+  name_prefix = "keycon"
```

Comment:
I'd just use `keyconjurer` if possible :) 

```suggestion
  name_prefix = "keyconjurer"
```

Reply:
You can't. :(

The maximum prefix length is 6.